src/main/java/gregtech/common/tileentities/machines/multi/GT_MetaTileEntity_MultiAutoclave.java

```diff
@@ -0,0 +1,384 @@
+                    Pair.of(GregTech_API.sBlockCasings11, 1),
+                    Pair.of(GregTech_API.sBlockCasings11, 2),
+                    Pair.of(GregTech_API.sBlockCasings11, 3),
+                    Pair.of(GregTech_API.sBlockCasings11, 4),
+                    Pair.of(GregTech_API.sBlockCasings11, 5),
+                    Pair.of(GregTech_API.sBlockCasings11, 6),
+                    Pair.of(GregTech_API.sBlockCasings11, 7)),
+                -2,
+                GT_MetaTileEntity_MultiAutoclave::setItemPipeTier,
+                GT_MetaTileEntity_MultiAutoclave::getItemPipeTier))
+        .addElement(
+            'F',
+            withChannel(
+                "coil",
+                ofCoil(GT_MetaTileEntity_MultiAutoclave::setCoilLevel, GT_MetaTileEntity_MultiAutoclave::getCoilLevel)))
+        .build();
+
+    @Override
+    public IStructureDefinition<GT_MetaTileEntity_MultiAutoclave> getStructureDefinition() {
+        return STRUCTURE_DEFINITION;
+    }
+
+    @Override
+    protected GT_Multiblock_Tooltip_Builder createTooltip() {
+        GT_Multiblock_Tooltip_Builder tt = new GT_Multiblock_Tooltip_Builder();
+        tt.addMachineType("Autoclave")
+            .addInfo("Controller Block for the Industrial Autoclave.")
+            .addInfo("Gains 12 parallels per item pipe casing tier.")
+            .addInfo("Each pipe casing (bronze, steel, titanium, tungstensteel)")
+            .addInfo("decreases the EU usageby 1/pipe tier.")
+            .addInfo("Heating Coils increase speed by 1/((tier + 1) / 2).")
+            .addInfo(AuthorVolence)
+            .addSeparator()
+            .beginStructureBlock(7, 5, 5, true)
+            .addController("Front Center")
+            .addCasingInfoMin("Pressure Containment Casings", 1, false)
+            .addCasingInfoExactly("Item Pipe Casings", 7, true)
+            .addCasingInfoExactly("Pipe Casings", 14, true)
+            .addCasingInfoExactly("Heating Coils", 7, true)
+            .addCasingInfoExactly("PTFE Frame", 42, false)
+            .addInputBus("Any of the Pressure Containment Casings", 1)
+            .addOutputBus("Any of the Pressure Containment Casings", 1)
+            .addEnergyHatch("Any of the Pressure Containment Casings", 1)
+            .addMaintenanceHatch("Any of the Pressure Containment Casings", 1)
+            .addMufflerHatch("Any of the Pressure Containment Casings", 1)
+            .toolTipFinisher("GregTech");
+        return tt;
+    }
+
+    private int mCasingAmount;
+
+    private void onCasingAdded() {
+        mCasingAmount++;
+    }
+
+    @Override
+    public boolean checkMachine(IGregTechTileEntity aBaseMetaTileEntity, ItemStack aStack) {
+        fluidPipeTier = -2;
+        itemPipeTier = -2;
+        mCasingAmount = 0;
+        mEnergyHatches.clear();
+        setCoilLevel(HeatingCoilLevel.None);
+        if (!checkPiece(STRUCTURE_PIECE_MAIN, 3, 6, 0)) return false;
+        // if (mCasingAmount < 8) return false;
```

Comment:
```suggestion
       if (mCasingAmount < x) return false;
```
Need to enable a minimum casing here as well as adjust the tooltip

Reply:
Adjusted, now needs minimum of 128 of the blocks